workshops/hcload/README.md

```diff
@@ -0,0 +1,465 @@
+---
+name: Hackclub CDN Uploader
+description: Make a CLI to upload to the Hackclub CDN and implement uploading to the CDN in your apps
+author: '@KhushrajRathod'
+img: https://cloud-bb4wyl0oo.vercel.app/0screenshot_2020-11-11_at_10.00.26_pm.png
+---
+
+# Hackclub CDN Uploader 
+
+<video autoplay loop muted>
+    <source src="https://cloud-7fq4zemw0.vercel.app/0hcloaddemo.mp4" type="video/mp4">
+</video>
+
+Everyone likes unlimited storage, especially when it's available as a cdn (i.e. you can directly embed items uploaded into webpages). If you're on the [Hack Club Slack](https://hackclub.com/slack/), you've probably uploaded something or the other to [#cdn](https://hackclub.slack.com/archives/C016DEDUL87) (If you aren't on the Slack / Haven't uploaded to #cdn, you should check both out!)
```

Comment:
Remove the (If you aren't on Slack...) stuff. Assume people know what the Hack Club Slack is, or that their club leader will explain it to them